drivers/spi/spi_shell.c

```diff
@@ -135,6 +139,30 @@ static int cmd_spi_conf(const struct shell *ctx, size_t argc, char **argv)
 	return 0;
 }
 
+static int cmd_spi_conf_cs(const struct shell *ctx, size_t argc, char **argv)
+{
+	struct device *dev = (struct device *)shell_device_get_binding(argv[CS_ARGV_GPIO_DEV]);
+
+	if (dev == NULL) {
+		shell_error(ctx, "device %s not found.", argv[CS_ARGV_GPIO_DEV]);
+		return -ENODEV;
+	}
+
+	uint32_t pin = strtol(argv[CS_ARGV_GPIO_PIN], NULL, 10);
+
+	config.cs.gpio.port = dev;
+	config.cs.gpio.pin = pin;
+
+	/* Include flags if provided */
+	if (argc == (CS_ARGV_GPIO_FLAGS + 1)) {
+		uint32_t flags = strtol(argv[CS_ARGV_GPIO_FLAGS], NULL, 16);
```

Comment:
It would be a good idea to validate the result of `strtol` when converting the pin number from a string to an integer.

Reply:
Sounds good. I've added parameter validation for both the Pin Number and the Flags.